skactiveml/pool/multi/_wrapper.py

```diff
@@ -0,0 +1,335 @@
+            Candidate samples from which the strategy can select.
+        A_cand : array-like, shape (n_samples, n_annotators), optional
+        (default=None)
+            Boolean matrix where `A_cand[i,j] = True` indicates that
+            annotator `j` can be selected for annotating sample `X_cand[i]`,
+            while `A_cand[i,j] = False` indicates that annotator `j` cannot be
+            selected for annotating sample `X_cand[i]`. If `A_cand is None`, each
+            annotator is assumed to be available for labeling each sample.
+            `A_cand` must only be None, if `n_annotators is non None`.
+        query_params_dict : dict, default=None
+            Dictionary for the parameters of the query method.
+            If `dict` contains a value for `y`, it is transformed by
+            `y_aggregate`. If `query_params_dict is None` an empty
+            dictionary is used.
+        batch_size : 'adaptive'|int, optional (default=1)
+            The number of samples to be selected in one AL cycle. If 'adaptive'
+            is set, the `batch_size` is set to 1.
+        A_pref : array-like, shape (n_samples, n_annotators) or
+                  (n_annotators,) optional (default=None)
+            The preferred ranking of each annotator.
+            1.) If `A_pref` is of shape (n_samples, n_annotators) for each
+             sample `i` the value-annotators pair `(i, j)` is preferably picked
+             over the pair `(i, k)` if `A_pref[i, j]` is greater or
+             equal to `A_pref[i, k]`.
+            2.) If `A_pref` is of shape (n_annotators,) for each sample
+            `i` the value-annotators pair `(i, j)` is preferentially picked
+             over the pair `(i, k)` if `A_pref[j]` is greater or
+             equal to `A_pref[k]`.
+            3.) If `A_pref` is None, the annotators are chosen at random, with
+             a different distribution for each sample.
+        return_utilities : bool, optional (default=False)
+            If true, also returns the utilities based on the query strategy.
+        pref_annotators_per_sample : int, array-like, optional (default=1)
+        array-like, shape (k,), k <= n_samples
+            If `pref_annotators_per_sample` is an int, the value indicates
+            the number of annotators that are preferably assigned to a candidate
+            sample, if annotators can still be assigned to the given candidate
+            sample.
+            If `pref_annotators_per_sample` is an int array, the values of the
+            array are interpreted as follows. The value at the i-th index
+            determines the preferred number of annotators for the candidate
+            sample at the i-th index in the ranking of the batch. The last index
+            of the pref_annotators_per_sample array (k-1) indicates the
+            preferred number of annotators for all candidate sample at an index
+            greater of equal to k-1.
```

Comment:
To what refers the index `X_cand` or the ranking obtained after employing a single pool-based query strategy.